app/controllers/admin_controller.rb

```diff
@@ -131,7 +131,9 @@ def mark_comment_spam
     if current_user && (current_user.role == 'moderator' || current_user.role == 'admin')
       if @comment.status == 1
         @comment.spam
-        flash[:notice] = "Comment has been marked as spam."
+        user = @comment.author
+        user.ban
```

Comment:
What do you think should we moderate user or ban it ?

Reply:
Ban is right - moderation is for a code of conduct violation. But we should better explain this in documentation somewhere! For example the buttons shown on an admin's view of others' profiles is confusing - both are shown as options!